app/Services/GocardlessMapper.php

```
@@ -0,0 +1,106 @@
+
+            'source_iban' => $this->get($transaction, 'debtorAccount.iban'),
+            'target_iban' => $this->get($transaction, 'creditorAccount.iban'),
+
+            'amount' => $this->get($transaction, 'transactionAmount.amount', 0),
+            'currency' => $this->get($transaction, 'transactionAmount.currency', 'EUR'),
+            'booked_date' => $bookedDateTime,
+            'processed_date' => $valueDateTime,
+            'partner' => $this->get($transaction, 'creditorName') ??
+                        $this->get($transaction, 'debtorName') ??
+                        $this->get($transaction, 'remittanceInformationUnstructuredArray.0'),
+            'description' => implode(' ', $this->get($transaction, 'remittanceInformationUnstructuredArray', [])).
```

PHPStan (2.1.15)

Parameter #2 $array of function implode expects array|null, mixed given.


(argument.type)